SU-SOE/soe_profile
# D8CORE-2068: News & Person Accelerants

# READY FOR REVIEW

# Summary
- Adds stanford_news
- Adds stanford_person
- and all the minutiae that follow adding major functionality

# Needed By (Date)
- May 14th

# Urgency
- Super dooper high

# Steps to Test

1. Check out branch D8CORE-2068 in the LelandD8 stack
2. Do a full update and build `composer update && blt di`
3. Run tests and perform some linky clicky of creating content
4. Review user permissions
5. Review code

# Affected Projects or Products
- SOE 

# Associated Issues and/or People
- D8CORE-2068
- D8CORE-2017

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
